src/DocChecks.jl

```diff
@@ -77,11 +79,13 @@ end
 
 function allbindings(checkdocs::Symbol, mod::Module, out = Dict{Utilities.Binding, Set{Type}}())
     for (obj, doc) in meta(mod)
-        isa(obj, IdDict{Any,Any}) && continue
+        isa(obj, IdDict{Any,Any}) && continue # is this ever true?!
+        @assert obj isa Docs.Binding
```

Comment:
This bit actually confuses me.. are the keys to the `Docs.meta` dictionary ever not `Docs.Binding` objects? One option, just to be extra sure that it doesn't crash, could be to do `isa(obj, Docs.Binding) || continue` here.